server/src/server.ts

```diff
@@ -0,0 +1,40 @@
+const express = require('express')
+const request = express.Request
+const response = express.Response
+const mongoose = require('mongoose')
+const dotenv = require('dotenv')
+
+const app = express()
+
+app.use(express.urlencoded({ extended: false }))
+app.use(express.json())
+
+dotenv.config()
+
+const username: string = process.env.MONGO_USER!
```

Comment:
move these files to a config.ts file, which reads the .env, return a json object, and throws error when the env variable doesn't exist...this gonna crash because of the `!`